x-pack/plugins/cases/server/saved_object_types/migrations/utils.test.ts

```diff
@@ -1,229 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import { noneConnectorId } from '../../../common';
-import { createExternalService, createJiraConnector } from '../../services/test_utils';
-import { transformConnectorIdToReference, transformPushConnectorIdToReference } from './utils';
-
-describe('migration utils', () => {
```

Comment:
These tests were moved to `x-pack/plugins/cases/server/services/user_actions/transform.test.ts`